.trybe/requirements.json

```
@@ -2,82 +2,82 @@
   "requirements": [
     {
       "identifier": "test_validar_conteudo_do_arquivo_gerado",
-      "description": "Executar o método `analyse_log` deverá preencher corretamente os dados no arquivo `data/mkt_campaign.txt`",
+      "description": "1.1 - Será validado se ao executar o método `analyze_log` os dados são preenchidos de forma correta no arquivo `data/mkt_campaign.txt`",
```

```suggestion
      "description": "1.1 - Será validado se, ao executar o método `analyze_log`, os dados são preenchidos de forma correta no arquivo `data/mkt_campaign.txt`",
```

Sempre que tiver um `... se ao executar o método `lalalala` coloca entre vírgulas assim! Acho que é o correto. Tem alguns lugares pra ajustar isso!